tests/meta/demultiplex/test_demux_post_processing.py

```diff
@@ -509,7 +511,9 @@ def test_is_not_bcl2fastq_folder_structure(
     demux_post_processing_api.flow_cell_dir = not_bcl2fastq_folder_structure
 
     # WHEN checking if it is a bcl2fastq folder structure
-    is_bcl2fastq_folder_structure = demux_post_processing_api.is_bcl2fastq_demux_folder_structure()
+    is_bcl2fastq_folder_structure = demux_post_processing_api.is_bcl2fastq_demux_folder_structure(
+        flow_cell_name=""
+    )
```

Comment:
Not too fond of sending flow_cell_name="" but I don't know if the code actually improves by setting it as an optional in the input of is_bcl2fastq_demux_folder_structure.

Reply:
Me neither, but the argument is not optional - the directory fixtures were created with the flow cell name set as an empty string

Reply:
Letting the tests be for now.